src/lwip/standalone/TapInterface.c

```diff
@@ -197,7 +197,7 @@ TapInterface_low_level_output(struct netif *netif, struct pbuf *buf)
     if (buf->tot_len > buf->len)
     {
         // Allocate a buffer from the buffer pool. Fail if none available.
-        outBuf = pbuf_alloc(PBUF_RAW, buf->tot_len + PBUF_LINK_ENCAPSULATION_HLEN, PBUF_POOL);
+        outBuf = pbuf_alloc(PBUF_RAW, buf->tot_len, PBUF_POOL);
```

Comment:
Just to confirm: the bug here was that for the tap device there is no additional link encapsulation like there was for WICED.

Reply:
Yep.  No bug, per se, just some confusing and unnecessary code/comments.